src/Microsoft.Health.Dicom.Api/Controllers/ChangeFeedController.cs

```diff
@@ -19,6 +19,8 @@
 
 namespace Microsoft.Health.Dicom.Api.Controllers
 {
+    [ApiController]
```

Comment:
This includes a couple other features detailed [here](https://docs.microsoft.com/en-us/aspnet/core/web-api/?view=aspnetcore-5.0#apicontroller-attribute). Is this a necessary change to use this API versioning library? Perhaps it also has functionality we want?

I see you also already disable the problem details, which I think is reasonable given we may not want to change our schemas

Reply:
Ah, I had assumed that it was required! Adding ` c.UseApiBehavior = false;` to the setup allows us to use ApiVersion without adding the controller attribute